### DISCUSSION TITLE
Transform / Translate make the popup (text) blur

### DISCUSSION THREAD

**Discussion by T-Nagui** [giorgiabosello/google-maps-react-markers#21] [Help]
Hi, 
I like the plugin for map but i have few issue and the main one is with this line
https://github.com/giorgiabosello/google-maps-react-markers/blob/d258557ab1c91326ef384032c725d8f80fe8d4ef/src/map/overlay.js#L35

The Transform and Translate make the text go blur and the best way is to use top and right instead of it

**Comment by giorgiabosello** [comment]
How come text become blur because of the transform?
This line is suggested directly from Google.

**Comment by T-Nagui** [comment]
hi 
in overlay the transform translate the pin and I have a custom pin and custom hover card 
the transfrom translate get the pin and card to the correct position and the blur bug is known bug that make the text get blur after 
https://css-tricks.com/forums/topic/blurry-text-in-chrome-and-opera-with-css-transform/

**Comment by giorgiabosello** [comment]
> hi 
> in overlay the transform translate the pin and I have a custom pin and custom hover card 
> the transfrom translate get the pin and card to the correct position and the blur bug is known bug that make the text get blur after 
> https://css-tricks.com/forums/topic/blurry-text-in-chrome-and-opera-with-css-transform/ 

I'm going to give it a check next week. :)

**Comment by T-Nagui** [comment]
ok Thinks Maybe i can make a pull request if i have a fixe if you don't mind

**Comment by giorgiabosello** [comment]
@T-Nagui yeah, sure.

**Comment by white-wolf97** [comment]
Hi Giorgia, I was thinking about contributing to the google-maps-react repository, but since that seems to not be mantained anymore, can I contribute in this project?

**Comment by giorgiabosello** [comment]
@white-wolf97 any help is appreciated. :)

**Comment by giorgiabosello** [comment]
@T-Nagui I'm not able to reproduce your issue. If you click on a marker [here](https://giorgiabosello.github.io/google-maps-react-markers/), you'll see text isn't blurred.
Please, provide a codesandbox example.
